src/base.iced

```diff
@@ -333,8 +353,7 @@ class Base
 
     @_v_customize_json ret
 
-    if @_v_include_pgp_hash()
-      await @full_pgp_hash {}, defer err, @_v_pgp_hash_dest(ret.body).full_pgp_hash
+    await @_add_pgp_details @_v_pgp_details_dest(ret.body), defer err, added
```

Comment:
Right now there's only one caller, so this isn't a big deal either way, but it might make more sense to have `_add_pgp_details` be responsible for knowing how to get the `details_dest`, rather than expecting all callers to do this properly.